### DISCUSSION TITLE
Way to use helm charts versioned as 0.0.0-rc0?

### DISCUSSION THREAD

**Discussion by jgz** [fluxcd/flux2#2044] [Q&A] [answered]
Is there a way to configure a helm sources/chart to allow versions for release candidates.?  

In helm you have to use the  --devel flag for these charts to show up in search.  is there a way to tell the helm source controller to do this?

**Comment by hiddeco** [comment, marked as answer]
Add a `-0` suffix to include pre-releases, e.g. `1.0.0-0`.

**Reply by jgz** [reply]
Thank you!
